pkg/ccl/backupccl/utils_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"context"
 	gosql "database/sql"
 	"fmt"
+	"github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backuputils"
```

Comment:
lint will probably shout at you, this import looks misplaced

Reply:
grrr `./dev lint short`